drivers/ethernet/Kconfig.mcux

```diff
@@ -43,8 +43,8 @@ config ETH_MCUX_RX_BUFFERS
 
```

Comment:
> Number of Tx buffers was reduced in pull #12583 for no reason.

That's not what I understood while reading #3132 

So do you get dumb_http_server sample working perfectly well with your change?